src/Common/Form/TitleType.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Common\Form;
+
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class TitleType extends TextType
+{
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        parent::configureOptions($resolver);
+
+        $resolver->setDefaults(
+            [
+                'label' => 'lbl.Title',
+                'required' => true,
```

Comment:
is required not always true by default?

Reply:
It is but I decided to explicitly define it because it was defined in some other places (like ContentBlocks) and it's also clearer for users.